### ISSUE TITLE
Python 3.12 and Parselmouth 0.5.0

### ISSUE THREAD

**Issue by YannickJadoul** [YannickJadoul/Parselmouth#107]
Just a tracking issue in case anyone is wondering where the new Python 3.12 wheels are.
There are still a couple of things on the list, which need to happen before releasing Parselmouth 0.5.0, mostly related to finishing the cleanup of the docs:
- [ ] Cleanup `docs/conf.py`
- [ ] Finish configuration of new theme
- [ ] Test/decide on Parselmouth favicon
- [ ] Test/decide SVG logos in docs
- [ ] Add all class docstrings
- [ ] Check int vs. float return from `parselmouth.praat.call`
- [ ] Double check locale-dependent functions after Praat update
- [ ] Add test on locale-independence?
- [ ] Update Praat
- [ ] Update libfmt
- [ ] Remove locale-independent workarounds in favor of scnlib

The actual wish list of new features is a lot longer, but those will unfortunately most likely have to wait for 0.6.0, as releasing Python 3.12 wheels should not be postponed for much more than a week.

Meanwhile, Python 3.12 is being tested in CI. Until new wheels are released, the [latest "Wheels" workflow build](https://github.com/YannickJadoul/Parselmouth/actions/workflows/wheels.yml?query=branch%3Amaster) should generate good-enough wheels to be installed until Parselmouth 0.5.0 gets released.

**Comment by aaron-randreth** [comment]
Hello,

Is there any update on python 3.12 support ? The last released wheels are still from 2022 and compiling from source seems to fail on all the machines I tried.

I hope you are doing alright, and thanks for the work ^^

**Comment by YannickJadoul** [comment]
Hi, @aaron-randreth, thanks for the kind bump!
I've come to terms that I should try to at least get the 3.12 wheels out as soon as possible, even if 0.5.0 isn't ready. The wheels are building on CI; I just need to make sure that I quickly run the tests as well...

**Comment by YannickJadoul** [comment]
Python 3.12 wheels for 0.4.3 are built, tested, and uploaded to PyPI. So installing with `pip` should now smoothly install binary pre-built wheels again!

Work on 0.5.0 continues!

**Comment by aaron-randreth** [comment]
Thanks, good luck for 0.5.0!
